player/video.c

```diff
@@ -59,6 +59,7 @@ static const char av_desync_help_text[] =
 "Audio/Video desynchronisation detected! Possible reasons include too slow\n"
 "hardware, temporary CPU spikes, broken drivers, and broken files. Audio\n"
 "position will not match to the video (see A-V status field).\n"
+"Consider trying `--profile=fast` and/or `--hwdec=auto-safe` as they may help.\n"
```

Comment:
Maybe get rid of the `--` as these are better set in `mpv.conf` instead of setting them in the cli everytime.

Reply:
We refer to option with `--` in all documents, so it seems consistent to keep it here too.

Reply:
For reference, this is the warning you get when using `vdpau`

![image](https://github.com/mpv-player/mpv/assets/79544946/0d2eb47b-0a6b-4e6c-b12b-1709184d335a)

(notice it says `vo=gpu` and not `--vo=gpu` etc).

Reply:
Note that `--profile=fast` still works in `mpv.conf` anyway.

Reply:
Seems to be one outlier. Few lines above you have with `--` https://github.com/mpv-player/mpv/blob/ef11d31c3acfd71307a94e44ad164a4861287675/video/out/vo_vdpau.c#L1039-L1042

I think `--` clearly denotes that we refer to the option and anyone who modified config knows what it means.